packages/caver-rpc/src/klay.js

```diff
@@ -2147,7 +2147,7 @@ class Klay {
                     logs: {
                         params: 1,
                         inputFormatter: [formatters.inputLogFormatter],
-                        subscriptionHandler: function(output) {
+                        subscriptionHandler: function (output) {
```

Comment:
Can you remove those modifications?
You can run `npm run lintFix` to fix linting.

Reply:
alright let me do that @jimni1222

Reply:
done @jimni1222